yarn.lock

```diff
@@ -4660,7 +4660,7 @@
     "@types/history" "*"
     "@types/react" "*"
 
-"@types/react@*", "@types/react@^17.0.0":
+"@types/react@*", "@types/react@17.0.0", "@types/react@^17.0.0":
```

Comment:
Should js updates be part of this PR?

 🤨 - 2 different react versions

Reply:
> Should js updates be part of this PR?
Somebody forgot to commit them before. I did not change anything just added missing files after running `yarn`.

> 🤨 - 2 different react versions

Here there is only one version of react: 17.0.0

Reply:
My bad. I could have sworn I saw two different entries for the actual lib (not the types) but I only see react@17.0.1, react@^17.0.1: 🤷